visualization/include/pcl/visualization/mouse_event.h

```diff
@@ -128,12 +128,12 @@ namespace pcl
         getSelectionMode () const;
 
       protected:
-        Type type_;
-        MouseButton button_;
-        unsigned int pointer_x_;
-        unsigned int pointer_y_;
-        unsigned int key_state_;
-        bool selection_mode_;
+        Type type_{};
+        MouseButton button_{};
+        unsigned int pointer_x_{0};
+        unsigned int pointer_y_{0};
+        unsigned int key_state_{0};
+        bool selection_mode_{false};
```

Comment:
```suggestion
        Type type_;
        MouseButton button_;
        unsigned int pointer_x_;
        unsigned int pointer_y_;
        unsigned int key_state_{0};
        bool selection_mode_;
```
Except for `key_state_`, they are all set in the constructor based on constructor parameters